lib/hexpm_web/templates/dashboard/tfa_auth_setup/index.html.eex

```diff
@@ -9,6 +9,10 @@
       <div class="panel-body recovery-codes-panel">
         <h1>Scan this barcode with your app</h1>
         <div style="text-align:center"><%= raw ViewHelpers.auth_qr_code_svg(@current_user) %></div>
+        <div>
+          You can also use your setup key (<%= @current_user.tfa.secret %>) to
+          manually configure your authenticator app
```

Comment:
```suggestion
          You can also use your setup key
          (<code id="tfa-secret" data-value="<%= @current_user.tfa.secret %>"><%= @current_user.tfa.secret %></code>
          <button type="button" class="copy-data-button" data-input-id="tfa-secret">📋</button>) to
          manually configure your authenticator app.
```

Demo:

<img width="755" alt="image" src="https://github.com/hexpm/hexpm/assets/76071/ffb8662d-3c10-4208-8add-f10354b5a108">

The copy button isn't perfectly designed but I think it's good enough. WDYT? cc @supersimple

Reply:
Great addition!